docs/methods/metrics/silhouette_score.md

```diff
@@ -0,0 +1,31 @@
+# Silhouette Score
+
+The silhouette score is calculated as the average of all points si, where si is the difference of minimum of inter cluster distance to average of intra cluster distance divided by maximum of both.
+The silhouette Score reflects how good the value of k is, where k is the number of clusters.
```

Comment:
how good the clusters are and not the value of k....as k is only for the case of kMeans but there exist other clustering algos